src/InterpLib/Error.ml

```diff
@@ -29,12 +29,15 @@ let report ?pos ~cls msg =
     | Warning -> "warning"
     | Note    -> "note"
   in
-  match pos with
-  | None ->
-    Printf.eprintf "%s: %s\n" name msg
-  | Some pos ->
+  match pos, Option.bind pos Position.get_text_range with
```

Comment:
Maybe we can match on `Option.map Position.get_text_range pos`? Its type is more precise.